app/src/main/kotlin/com/simplemobiletools/gallery/helpers/MediaSideScroll.kt

```diff
@@ -0,0 +1,150 @@
+package com.simplemobiletools.gallery.helpers
+
+import android.app.Activity
+import android.content.Context
+import android.media.AudioManager
+import android.os.Handler
+import android.provider.Settings
+import android.view.MotionEvent
+import android.widget.TextView
+import com.simplemobiletools.gallery.R
+import com.simplemobiletools.gallery.activities.ViewPagerActivity
+import com.simplemobiletools.gallery.extensions.audioManager
+
+class MediaSideScroll(val context: Context, val activity: Activity, val slideInfoView: TextView, val onSpecialEvent: () -> Unit) {
```

Comment:
Activity can just be cast to Context since it extends it, not sure what's better.

Reply:
ye, remove the context and leave the activity only. Also add an empty line at each files end